### PR TITLE
Project series

### PR DESCRIPTION
**Pull request by kellyyan**
Created project series page & updated font to Quicksand for the entire site.

### CODE REVIEW

File: pages/ProjectSeries.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import "../css/index.css";
+
+
+export default function ProjectSeriesPage() {
+  return (
+    <Layout setGoogleId={id => setGoogleId(id)}>
+      <div>
+        <BackgroundImage
+          source="Background/ocean.jpg"
+          missing_source="banner_1.png"
```
Comment (gmundewadi) [inline pages/ProjectSeries.js:13]: missing_source = “logo.png"

File: pages/Officers.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import Layout from "../components/Website_Layout/Layout";
+import BackgroundImage from "../components/Page_Layout/BackgroundImage";
+
+export default function ResourcesPage() {
+  return (
+    <Layout setGoogleId={id => setGoogleId(id)}>
+      <div>
+        <BackgroundImage
+          source="Background/sunset.jpg"
+          missing_source="banner_1.png"
```
Comment (gmundewadi) [inline pages/Officers.js:11]: missing_source=“logo.png"

**Review by gmundewadi — CHANGES_REQUESTED** [review]

**Comment by gmundewadi** [comment]
Also noticed that there was a build failure when deploying to QA

_ModuleNotFoundError: Module not found: Error: Can't resolve '../components/Page_Layout/Officers' in '/tmp/build_c357bf31_/pages'_

**Review by gmundewadi — APPROVED** [review]
lgtm

File: pages/Officers.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import Officers from "../components/Page_Layout/Officers";
```
[the remark this answers is not in the document]
Comment (kellyyan) [reply pages/Officers.js]: i just deleted the Officers page so it should be fixed

**Pull request merged**